### PR TITLE
Rework Style

### PR DESCRIPTION
**Pull request by Starkium**
Particles demo was causing issues for people, reworked this to be more compatible.

There's still an issue of this not aligning properly when scaling and likely on mobile devices. We need another pass on  this.

I'd be fine pushing out this current version since it alleviates the problem of the particles demo for now.

### CODE REVIEW

**Review by flare199 — APPROVED** [review]
Good as a temporary fix but we should optimize and clean it up later on

**Comment by sshiiden** [comment]
You should use the SVG file for a clearer image.

**PNG**
image
**SVG**
image

**Comment by Starkium** [comment]
> You should use the SVG file for a clearer image.
> 
> **PNG** image **SVG** image

I'm not a web dev so forgive me for potentially sounding dumb, but isn't svg more expensive?

**Comment by sshiiden** [comment]
For simple images like logos, SVG files are less expensive.

**Review by flare199 — APPROVED** [review]

File: static/css/effects.css
Comment (sshiiden) [inline static/css/effects.css:1]: The png is used instead of the svg.

**Review by sshiiden — APPROVED** [review]

File: static/css/effects.css
Comment (Starkium) [reply static/css/effects.css:1]: fixed, but sometimes the svg has an issue with aliasing when moving. I need to look into that later

**Review by Norrox — APPROVED** [review]

**Pull request merged**